_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "NewPipe 0.19.0 released: Group your feeds and mute them"
+short: "NewPipe 0.19.0 released"
+date: 2020-03-29 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+A new day, a new full release. In today's update, NewPipe is proudly bringing you a mute button and a great way to manage your subscribed channel, on top of the usual smaller improvements and fixes.<!-- more -->
+
+
+## Feed groups
```

Comment:
I know it says Feed Groups in the app itself, but I think calling them Channel Groups will be clearer, and the string should be changed in the app too. We're grouping channels _into_ feeds. @TobiGr
```suggestion
## Channel Groups
```

Reply:
>  We're grouping channels **into** feeds.

That's a good point. However, we can only change the app strings in the next update. They are already translated into several languages. Can you please open a PR / an issue to discuss this in the app repo or on IRC?

Reply:
No problem. I'll open an issue, not a PR, since B0pol is the expert on translations.

Reply:
So Im not going to merge the suggestion

Reply:
Yeah, let's leave this for the next one.